airgun/views/rhai.py

```diff
@@ -10,3 +10,20 @@ class AllRulesView(BaseLoggedInView):
     @property
     def is_displayed(self):
         return self.title.is_displayed and self.search.is_displayed
+
+
+class OverviewDetailsView(BaseLoggedInView):
+    title = Text(".//h1[normalize-space(.)='Overview']")
+    inventory = Table(".//h2[span[normalize-space(.)='Newest Systems']]/"
+                      "parent::section//table")
+    security_issues = Text(".//div[@ng-if='securityErrors']/div")
+    stability_issues = Text(".//div[@ng-if='stabilityErrors']/div")
+
+    @property
+    def is_displayed(self):
```

Comment:
that logic is invalid as I opened the page and don't have `security_issues` and `stability_issues` elements on the page, so doubtfully they are criteria for `is_displayed`